src/services/signatureHelp.ts

```diff
@@ -169,12 +169,12 @@ namespace ts.SignatureHelp {
             : { invocation: info.invocation.node, argumentCount: info.argumentCount, argumentIndex: info.argumentIndex };
     }
 
-    function getArgumentOrParameterListInfo(node: Node, sourceFile: SourceFile): { readonly list: Node, readonly argumentIndex: number, readonly argumentCount: number, readonly argumentsSpan: TextSpan } | undefined {
+    function getArgumentOrParameterListInfo(node: Node, position: number, sourceFile: SourceFile): { readonly list: Node, readonly argumentIndex: number, readonly argumentCount: number, readonly argumentsSpan: TextSpan } | undefined {
         const info = getArgumentOrParameterListAndIndex(node, sourceFile);
         if (!info) return undefined;
         const { list, argumentIndex } = info;
 
-        const argumentCount = getArgumentCount(list);
+        const argumentCount = getArgumentCount(list, isInString(sourceFile, position, node));

```

Comment:
```suggestion
        const argumentCount = getArgumentCount(list, /*ignoreTrailingComma*/ isInString(sourceFile, position, node));
```

Reply:
Maybe alternatively
```suggestion
        const argumentCount = getArgumentCount(list, /*trailingCommaAffectsArity*/ 
 !isInString(sourceFile, position, node));
```

with the same corresponding changes elsewhere